PowerEditor/src/Parameters.h

```diff
@@ -1734,4 +1736,5 @@ class NppParameters final
 	void initScintillaKeys();	//these functions have to be called first before any modifications are loaded
 	int getCmdIdFromMenuEntryItemName(HMENU mainMenuHadle, generic_string menuEntryName, generic_string menuItemName); // return -1 if not found
 	int getPluginCmdIdFromMenuEntryItemName(HMENU pluginsMenu, generic_string pluginName, generic_string pluginCmdName); // return -1 if not found
+	winVer getWindowsVersion();

```

Comment:
No, it can't be as it sets value for member _platForm in switch statement below.